app/partials/addMastersModal.html

```diff
@@ -0,0 +1,16 @@
+<div class="modal-container" ng-click="unfocusModal($event)">
+    <div class="modal {{vtClass}}" ng-click="$broadcast('onParentClick', $event)" ng-keydown="$broadcast('keyDown', $event)" tabindex="0" autofocus>
+        <div class="close-modal" ng-click="$emit('closeModal')"></div>
+        <h2>Add Masters</h2>
+        <form>
+        	<list-view items="availableMasters" list-class="checkbox-list">
```

Comment:
What the whitespace?  👾 

We're using 4 space indentation in this repo.  Perhaps we should add an .editorconfig file back to the repo to make this style easier to adhere to?